### PR TITLE
Added README.md for the Project

### PR DESCRIPTION
**Pull request by Error042**
Added a README.md for description of the project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Netflix AutoSkip
+
+## What is this?
+This is a simple plugin for Chromium based and Firefox based browsers that skips the following Netflix screens:
+
+- The "Who's watching?" screen
+- The "Skip Intro" button
+- The "Continue Watching" button
+
+## How to install
```
Comment (McChronicle) [inline README.md:10]: maybe put the instructions below under a "install manually" section, as the standard way for installing should probaply be via the browser extension stores
```suggestion
## How to install
### Chromium based browsers
1. Visit the page for this extension [in the chrome web store](https://chrome.google.com/webstore/detail/netflix-autoskip/oglhelhnbhiigifcgohdmhgaodaonhjl)
2. Click "add"
3. Enjoy! Updates will be applied automatically as we publish them on the chrome web store.

### Firefox based browsers
1. Visit the page for this extension [in the firefox addon store](https://addons.mozilla.org/de/firefox/addon/netflix-autoskip/)
2. Click "add to firefox"
3. Enjoy! Updates will be applied automatically as we publish them on the firefox addon store.

### Manual installation
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+# Netflix AutoSkip
+
+## What is this?
+This is a simple plugin for Chromium based and Firefox based browsers that skips the following Netflix screens:
+
+- The "Who's watching?" screen
+- The "Skip Intro" button
+- The "Continue Watching" button
```
Comment (McChronicle) [inline README.md]: this extension doesn't skip the "who's watching" screen, but it auto clicks "next episode"
```suggestion
This is a simple plugin for Chromium based and Firefox based browsers that skips the following Netflix screens:

- The "Skip Intro" button
- The "Continue Watching" button
- The "Next Episode" button (credits)
```

**Review by McChronicle — CHANGES_REQUESTED** [review]
looks nice, only some smaller suggestions

**Comment by Error042** [comment]
@McChronicle I updated the file by your suggestions

**Review by McChronicle — APPROVED** [review]

**Pull request merged**